server/models/migrations/next/migrate.sql

```diff
@@ -181,3 +181,20 @@ ALTER TABLE `account_reference` MODIFY COLUMN `description` VARCHAR(200) NOT NUL
  * @desc: stock setting for cost center to use in case of stock loss
  */
 ALTER TABLE `stock_setting` ADD COLUMN `default_cost_center_for_loss` MEDIUMINT(8) NULL;
+
+/**
+ * @author: mbayopanda
+ * @desc: fix allocation bases link and report
+ */
+UPDATE `unit` SET `path` = '/cost_center/allocation_bases' WHERE id = 299;
+INSERT INTO `report` (`report_key`, `title_key`) VALUES
+  ('cost_center_step_down', 'TREE.COST_CENTER_STEPDOWN');
+
+/**
+ * @author: mbayopanda
+ * @desc: add predefined allocation basis
+ */
+INSERT INTO `cost_center_allocation_basis` (`id`, `name`, `units`, `description`, `is_predefined`) VALUES
```

Comment:
Note: If PR #5947 gets merged first, this will need a little adjusting since I added items 1-3 in my version of migrate.sql.  I suggest adding these (4-6) into the previous insert (with 1-3) above.

ALSO, please add the SQL 'IGNORE' keyword in your 'INSERT' statements.